scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/processor/DefaultPacBio.java

```diff
@@ -246,6 +256,7 @@ protected StatusResponse getStatus(String url) {
 
   @Override
   public NotificationDto process(File runDirectory, TimeZone tz) throws IOException {
+    this.timeZone = tz;
```

Comment:
Don't put this is in a field. We've already had concurrency issues this with. Pass it as a parameter to the relevant methods.